Casks/j/jslegendre-themeengine.rb

```diff
@@ -9,10 +9,13 @@
 
   livecheck do
     url :url
-    strategy :git do |tags|
+    regex(/^v?(\d+(?:\.\d+)*)\((\d+)\)$/i)
```

Comment:
```suggestion
    regex(/^v?(\d+(?:\.\d+)+)(?:\((\d+)\))?$/i)
```

The upstream repository also has some older tags like `0.0.4` (from before the fork), so it probably wouldn't hurt to make the parenthetical suffix optional.

There aren't any tags that use only one number, so we should use the default `+` form of the standard regex (i.e., it's less likely to match something unexpected in the future).